L.TileLayer.PouchDBCached.js

```diff
@@ -139,28 +139,29 @@ L.TileLayer.include({
 	// Use with care! This can spawn thousands of requests and
 	//   flood tileservers!
 	seed: function(bbox, minZoom, maxZoom) {
+		if (!this.options.useCache) return;
 		if (minZoom > maxZoom) return;
 		if (!this._map) return;
 
 		var queue = [];
 
-		for (var z = minZoom; z<maxZoom; z++) {
+		for (var z = minZoom; z<=maxZoom; z++) {
 
 			var northEastPoint = this._map.project(bbox.getNorthEast(),z);
 			var southWestPoint = this._map.project(bbox.getSouthWest(),z);
 
 			// Calculate tile indexes as per L.TileLayer._update and
 			//   L.TileLayer._addTilesFromCenterOut
-			var tileSize   = this._getTileSize();
+			var tileSize = this.getTileSize(); //_getTileSize(), used to return an integer?
```

Comment:
This is because Leaflet 1.0 introduced an API change to support non-square tiles, so `getTileSize()` returns a `L.Point` with the tile size. The `y` coordinate of `tileBounds` three lines down should rely on the `y` size of the tile.